html/index.html

```diff
@@ -0,0 +1,452 @@
+        
+                        <div id="javascript">
+                            
+                            <span> JavaScript</span>
+        
+                        </div>
+        
+                    </li>
+        
+                    <li class="language-html">
+        
+                        <img src="../assets/h5Logo.png" alt="">
+        
+                        <div id="html">
+                            
+                            <span> HTML</span>
+        
+                        </div>
+        
+                    </li>
+        
+                    <li class="language-css">
+        
+                        <img src="../assets/css3Logo.png" alt="">
+        
+                        <div id="css"> 
+                            
+                            <span> CSS </span>
+        
+                        </div>
+        
+                    </li>
+        
+                </ul>
+        
+                <ul class="language-footer">
+        
+                    <li class="language-framwork">
+        
+                        <p class="bold-text">Frameworks</p>
+        
+                        <img src="../assets/dropdown.png" alt="" class="dropside-img">
+            
+                    </li>
+        
+                    <hr id="hr">
+        
+                    <li class="language-skill">
+        
+                        <p  class="bold-text">Skills</p>
+        
+                        <img src="../assets/dropdown.png" alt="" class=" dropside-img">
+            
+                    </li>
+        
+                    <hr>
+        
+        
+        
+                </ul>
+        
+        
+            </ul>
+        
+            <div class="section">
+        
+                <div class="contact-me-section">
+        
+                    <p id="contact-me">Contact me</p>
+        
+                    <p id="contact-msg"> 
+                        If you have an application you are 
+                        interested in developing, a feature 
+                        that you need built or a project that
+                        needs coding. I’d love to help with it 
+                    </p>
+        
+                </div>
+        
+        
+                <div class="form-section">
+        
+                    <form action="">
+        
+                        <input type="text" placeholder="Toyyib" class="text"> <br>
+                        <input type="email" placeholder="Simpleshaikh2@gmail.com" class="text"> <br>
+                        <textarea type="text" placeholder="write your message heres" id="type-msg"> </textarea><br>
+        
+                    </form>
+        
+                    <div class="btn-sec">
+        
+                        <button class="sec-btn">   
+                            Get in touch
+                        </button>
+            
+                        <span class="dot"></span>
+        
+                    </div>
```

Comment:
- [ ] While you did a great job implementing the contact form section, I can not review it because it's not part of this milestone :cry: